lms/djangoapps/discussion/views.py

```diff
@@ -832,11 +833,11 @@ def course_discussions_settings_handler(request, course_key_string):
         'divided_inline_discussions': divided_inline_discussions,
         'divided_course_wide_discussions': divided_course_wide_discussions,
         'always_divide_inline_discussions': discussion_settings.always_divide_inline_discussions,
-        'division_scheme': discussion_settings.division_scheme
+        'division_scheme': discussion_settings.division_scheme,
+        'available_division_schemes': available_division_schemes(course_key)
```

Comment:
This is my current thought on how the instructor dashboard discussion tab should work:

1) Always render radio buttons for both cohort and enrollment track schemes.
2) If a scheme is not in available_division_schemes AND it is not the current division_scheme, hide that radio button (display none).
3) In the case where cohorts was enabled and cohort was the selected division_scheme, if staff disables cohorts, keep cohort as the selected division_scheme (and visible). Discussions will still not be divided due to this logic: https://github.com/edx/edx-platform/pull/15037/files#diff-d1b71fc1244cf687d2caac1c116f208bR863
4) If cohorts was enabled when the instructor tab first rendered and cohort is NOT the selected division_scheme, hide the cohort scheme radio button if course staff disables cohorts.